### PR TITLE
docs: adding env management using Workspaces

### PR DESCRIPTION
**Pull request by AnnuCode**

### CODE REVIEW

File: how-to/manage-environments/using-workspaces.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Using Workspaces
+description: Learn how to manage environments using Workspaces.
```
Comment (soerenmartius) [inline how-to/manage-environments/using-workspaces.md:3]:
```suggestion
title: Using Terraform CLI Workspaces | How-to Guide
description: Learn how to manage environments using Terraform CLI Workspaces with Terramate.

```

File: how-to/manage-environments/using-workspaces.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+cd stacks/bob
+
+terraform workspace list
+
+Sync the default workspace as the default target to Terramate Cloud:
+
+terramate run \               
+  --sync-drift-status \
+  --terraform-plan-file deploy.tfplan \
+  --target default \
```
Comment (soerenmartius) [inline how-to/manage-environments/using-workspaces.md:36]: @AnnuCode when targets are enabled, does a user always need to explicitly set the default target or can it be omitted?

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: how-to/manage-environments/using-workspaces.md (hunk 2 above)
Comment (AnnuCode) [reply how-to/manage-environments/using-workspaces.md:36]: We need to use `--target` when targets are enabled and when we are using any `--sync` option.

**Review by soerenmartius — APPROVED** [review]
LGTM

**Pull request merged**